src/PowerLincModem.ts

```diff
@@ -889,7 +894,7 @@ export default class PowerLincModem extends EventEmitter2 {
 		this.once(
 			['p', PacketID.StandardMessageReceived.toString(16), MessageSubtype.BroadcastMessage.toString(16), toAddressString(deviceID)],
 			(data: Packet.StandardMessageRecieved) =>
-				resolve(PowerLincModem.getDeviceInfo(data.to[0],data.to[1]))
+				resolve(PowerLincModem.getDeviceInfo(data.to[0],data.to[1],data.to[2]))
```

Comment:
Including firmware byte (`to[0]` is cat, `to[1]` is subcat and `to[3]` is firmware version)

Reply:
Gotcha